sbom-utility-scripts/scripts/add-image-reference-script/README.md

```diff
@@ -0,0 +1,146 @@
+```
+The script stores the updated SBOM in the output path provided.
+
+## List of updates
+### SPDX
+The script updates the following fields in the SPDX SBOM:SHA256
+- `packages` - the script adds a new package with the image reference information
+- `relationships` - the script adds a new relationship between the package and the image reference
+- `name` - the script adds the image reference as a name
+
+The logic of adding new packages and updating replationships is visualized in the following diagram:
+```
+cachi2 SPDX SBOM:
+
+                ROOT
+             /        \
+            /          \
+        DESCRIBES   DESCRIBES
+          /              \
+         /                \
+<pip main package>  <npm main package>
+        |                  |
+        |                  |
+     CONTAINS           CONTAINS
+        |                  |
+        |                  |
+    <pip deps>         <npm deps>
+
+
+SBOM after enrichment:
+
+                ROOT
+                 |
+                 |
+              DESCRIBES
+                 |
+                 |
+             <container>
+             /        \
+            /          \
+        CONTAINS    CONTAINS
+          /              \
+         /                \
+<pip main package>  <npm main package>
+        |                  |
+        |                  |
+     CONTAINS           CONTAINS
+        |                  |
+        |                  |
+    <pip deps>         <npm deps>
+
+```
+
+#### Example
+```json
+{
+  "name": "quay.io/foo/bar/ubi8@sha256:011ff0cd8f34588d3eca86da97619f7baf99c8cc12e24cc3a7f337873c8d36cc",
+  "packages": [
+    {
+      "SPDXID": "SPDXRef-image",
+      "name": "ubi8",
+      "versionInfo": "1.1",
+      "downloadLocation": "NOASSERTION",
+      "licenseConcluded": "NOASSERTION",
+      "supplier": "NOASSERTION",
+      "externalRefs": [
+        {
+          "referenceLocator": "pkg:oci/ubi8@sha256:011ff0cd8f34588d3eca86da97619f7baf99c8cc12e24cc3a7f337873c8d36cc?repository_url=quay.io/foo/bar/ubi8",
+          "referenceType": "purl",
+          "referenceCategory": "PACKAGE-MANAGER"
+        }
+      ],
+      "checksums": [
+        {
+          "algorithm": "SHA256",
+          "checksumValue": "011ff0cd8f34588d3eca86da97619f7baf99c8cc12e24cc3a7f337873c8d36cc"
+        }
+      ]
+    },
+  ],
+  "relationships": [
+    {
+      "spdxElementId": "SPDXRef-DOCUMENT",
+      "relationshipType": "DESCRIBES",
+      "relatedSpdxElement": "SPDXRef-image"
+    },
+  ]
+}
+
+```
+
+### CycloneDX
+- `components` - the script adds a new component with the image reference information
+- `metadata.component` - the script adds the image reference as a metadata.component
+
+#### Example
+```json
+{
+  "metadata": {
+    "component": {
+      "type": "container",
+      "bom-ref": "pkg:oci/ubi8@sha256:011ff0cd8f34588d3eca86da97619f7baf99c8cc12e24cc3a7f337873c8d36cc?repository_url=quay.io/foo/bar/ubi8",
```

Comment:
nitpick: the README shouldn't include `bom-ref` anymore either

Reply:
Removed